bento_beacon/config_files/config.py

```diff
@@ -1,9 +1,24 @@
 import json
 import os
+import urllib3
+
 
 GA4GH_BEACON_REPO_URL = "https://raw.githubusercontent.com/ga4gh-beacon/beacon-v2"
 
 
+def str_to_bool(value: str) -> bool:
+    return value.strip().lower() in ("true", "1", "t", "yes")
+
+
+DEBUG = str_to_bool(os.environ.get("BENTO_DEBUG", os.environ.get("FLASK_DEBUG", "false")))
```

Comment:
I deliberately didn't have a config value just called "DEBUG" because this value is set internally by Flask and [setting it in config is discouraged](https://flask.palletsprojects.com/en/2.3.x/config/#debug-mode).

Also, I wonder if a more clear setting wouldn't be something like 

`BEACON_DEBUG = DEBUG or BENTO_DEBUG`

Reply:
yeah, this is just a prefix-less bento debug in reality (and thus why it is outside of the config - only to be used to suppress the annoying urllib3 warning spam), so I may just rename it to that...

Reply:
thanks for killing the spam!